emeis/oidc_auth/tests/test_authentication.py

```diff
@@ -89,7 +89,7 @@ def test_authentication_new_user(
     except AuthenticationFailed:
         assert not create_user
     else:
-        assert user.email == email
+        assert user.user.email == email
```

Comment:
Wouldn't it be cleaner to write a factory function instead of a class, so that we don't get `user.user`?